server/src/main/java/org/jboss/as/server/controller/resources/SystemPropertyResourceDefinition.java

```diff
@@ -69,27 +70,35 @@
             .setAllowExpression(true)
             .build();
 
+    public static final SimpleAttributeDefinition PARSE_TIME = SimpleAttributeDefinitionBuilder.create(ModelDescriptionConstants.PARSE_TIME, ModelType.BOOLEAN, true)
+            .setValidator(new ModelTypeValidator(ModelType.BOOLEAN, true))
+            .setDefaultValue(new ModelNode(false))
+            .setAllowExpression(false)
+            .build();
+
     static final AttributeDefinition[] ALL_ATTRIBUTES = new AttributeDefinition[] {VALUE, BOOT_TIME};
-    static final AttributeDefinition[] SERVER_ATTRIBUTES = new AttributeDefinition[] {VALUE};
+    static final AttributeDefinition[] SERVER_ATTRIBUTES = new AttributeDefinition[] {VALUE, PARSE_TIME};
```

Comment:
A reason to drop this attribute and always try to set as David suggested is it's really only valid for standalone servers, not domain servers. For domain it's cruft.